authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationConfig.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.authorization.ranger;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.gravitino.authorization.Privilege;
+
+/**
+ * Ranger authorization use this configuration to mapping Gravitino privilege to the Ranger
```

Comment:
Change class name from`RangerAuthorizationConfig` to `PrivilegeMappingRules`?
and change the function name from 
1. `initializePrivilegesMappingConfig()` -> `setPrivilegesMappingRule()`
2. `initializeOwnerPrivilegesConfig()` -> `setOwnerPrivilegesRule()`
3. `initializePolicyResourceDefinesConfig()` -> `setPolicyResourceDefinesRule()`
What do you think?

Reply:
Typically, it is hard to design an interface for like "initializeXXX" or "setXXX", because it is hard for developers to know how to correctly implement a code about "initializeXXX" or "setXXX". Instead, I would suggest like:

```
interface RangerPrivilegesMappingProvider {

Map<GravitinoPrivilege, Set<RangerPrivileges>> privilegesMappingRule()
Map<GravitinoPrivilege, Set<RangerPrivileges>> ownerMappingRule()
...
}
```
Just for your reference, please take a look, thanks.

Reply:
OK, I accept your suggestion.